.github/workflows/CI.yml

```diff
@@ -20,7 +20,6 @@ jobs:
       matrix:
         version:
           - '1'
-          - '1.6'
```

Comment:
Could still be useful to keep testing the oldest compatible version explicitly since 1.10 will be released soon?

Reply:
I guess. I think we'll just need to update a lot of package CIs anyways. We'll want to just bump things to v1.10 since that's the suggested LTS.

Reply:
Yeah I'm not going to worry about this right now, but you can put a PR if you want to do that. I think when v1.10 comes out, since that's poised to be right after a huge round of a major bumps, I think we'll just bump packages to v1.10 and backport what's needed, and then hopefully that's the new LTS.